### ISSUE TITLE
[Bug]: collection load error 

### ISSUE THREAD

**Issue by zxcvjl** [milvus-io/milvus#19717]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.1.1
- Deployment mode(standalone or cluster): standalone 
- SDK version(e.g. pymilvus v2.0.0rc2):pymilvus 2.1.1
- OS(Ubuntu or CentOS): 
- CPU/Memory: 48c/182G
- GPU: 
- Others:
```

### Current Behavior

every time collection is called, load is required. After use, collection release
then ,today when i call ,
Collection. load() failed  

### Steps To Reproduce

```markdown
from pymilvus import connections, FieldSchema, CollectionSchema, DataType, Collection, utility
host = '10.99.46.106'  
port = '19530'
connections.connect(host= host, port= port)
print(utility.list_collections())
search_collection1 = Collection(name="WBP_Images")
search_collection1.load()
```

### Milvus Log

 ["show collection failed"] [role=querycoord] [collectionID=435557063674560513] [msgID=436611607790354439] [error="collection 435557063674560513 has not been loaded to memory or load failed"]
[2022/10/12 02:18:55.370 +00:00] [ERROR] [proxy/task_scheduler.go:468] ["Failed to execute task: collection WBP_Images has not been loaded to memory or load failed"] [traceID=147af6e9e2f410e8] [stack="github.com/milvus-io/milvus/internal/proxy.(*taskScheduler).processTask\n\t/go/src/github.com/milvus-io/milvus/internal/proxy/task_scheduler.go:468\ngithub.com/milvus-io/milvus/internal/proxy.(*taskScheduler).definitionLoop\n\t/go/src/github.com/milvus-io/milvus/internal/proxy/task_scheduler.go:494"]
[2022/10/12 02:18:55.370 +00:00] [WARN] [proxy/impl.go:868] ["ShowCollections failed to WaitToFinish"] [error="collection WBP_Images has not been loaded to memory or load failed"] [role=proxy] [MsgID=436611607790354439] [DbName=] [TimeStamp=0] [ShowType=InMemory] [CollectionNames="[WBP_Images]"]
[2022/10/12 02:18:55.613 +00:00] [WARN] [grpclog/grpclog.go:46] ["[transport]transport: http2Server.HandleStreams failed to read frame: read tcp 172.18.0.4:19530->10.99.46.31:50540: read: connection reset by peer"]

**Comment by yanliang567** [comment]
@zxcvjl it seems to to a known issue, which was fixed on 2.1.4. could you please retry on 2.1.4?
Also please make sure you have enough memory on querynode for collection loading.

/assign @zxcvjl 
/unassign

**Comment by wg1026688210** [comment]
> 
@yanliang567 @zxcvjl 
mark,we encountered the same problem at master 

git reset --hard 0343a8c62
